docs/source/roles/system_manager/manage_users.md

```diff
@@ -183,6 +183,18 @@ A sample email might look like the following
 >
 > --details about network and location/VPN restrictions here--
 
+(modifying_network)=
+
+## {{globe_with_meridians}} Manually modifying network restrictions
+
+One of the controls used by Tier 2/3 SREs is to restrict access based on network addresses. If the users were to require access from different IP addresses - if their institutional IP address changes, or an additional location with a different network were approved - then the network addresses that are allowed access can be modified manually.
+
+- In the Azure Portal, navigate to `RG_SHM_<SHM_ID>_SRE_<SRE_ID>_NETWORKING`
+- On the `Overview` tab, navigate to the Network Security Group for the appropriate remote desktop service (e.g `NSG_SHM_<SHM_ID>_SRE_<SRE_ID>_GUACAMOLE`)
+- Navigate to `Inbound Security Rules`, and open the entry called `AllowUsersApprovedHttpsInbound`
+- For the field `Source IP addresses/CIDR ranges`, add the desired IP address or range to the existing entry, or overwrite the existing entry if it is no longer required
+- Users will then be able to access the remote desktop interface from the new IP address
```

Comment:
It might be worth rephrasing these to say something like "Update the Source IP addresses/CIDR ranges field to include the IP addresses (and only the IP addresses) that should be able to access the SRE" and "Users will now be able to access the remote desktop from only the desired IP addresses".

That way we emphasise that this should _only_ contain approved IP addresses, rather than being a list which we repeatedly append to.

Reply:
Rather spell it out a bit more than say "include only IP addresses", as that implies there are IP addresses that need to be removed, which may not be the case. 

```suggestion
- Update the `Source IP addresses/CIDR ranges` field to include IP addresses that should be able to access the SRE and remove any that should not
- Users will now be able to access the remote desktop interface from only the desired IP addresses
```
```suggestion
- Update the `Source IP addresses/CIDR ranges` field to include IP addresses that should be able to access the SRE and remove any that should not
- Users will now be able to access the remote desktop interface from only the desired IP addresses
```

Reply:
Sounds good to me. Both are equivalent but we should go for whatever is clearer. I think you are right that the process here is 'making a change' so it makes sense to present that as add/remove rather than rewrite from scratch.

(This is kind of the same as the declarative vs imperative)